sbt/src/sbt-test/project/unified/build.sbt

```diff
@@ -0,0 +1,12 @@
+lazy val root = (project in file("."))
+  .settings(
+    Global / cancelable := true,
+    ThisBuild / scalaVersion := "2.12.2",
```

Comment:
As a non-expert sbt user, this is much more obvious to me that there is a hierarchy of settings and I am setting one in a specific place in the hierarchy 👍